implants/lib/eldritch/src/file.rs

```diff
@@ -83,92 +90,201 @@ struct File {
-        Ok(NoneType{})
+        Ok(NoneType {})
     }
     fn copy(this: FileLibrary, src: String, dst: String) -> anyhow::Result<NoneType> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         copy_impl::copy(src, dst)?;
-        Ok(NoneType{})
+        Ok(NoneType {})
     }
     fn compress(this: FileLibrary, src: String, dst: String) -> anyhow::Result<NoneType> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         compress_impl::compress(src, dst)?;
-        Ok(NoneType{})
+        Ok(NoneType {})
     }
     fn download(this: FileLibrary, uri: String, dst: String) -> anyhow::Result<NoneType> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         download_impl::download(uri, dst)?;
-        Ok(NoneType{})
+        Ok(NoneType {})
     }
     fn exists(this: FileLibrary, path: String) -> anyhow::Result<bool> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         exists_impl::exists(path)
     }
     fn hash(this: FileLibrary, path: String) -> anyhow::Result<String> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         hash_impl::hash(path)
     }
     fn is_dir(this: FileLibrary, path: String) -> anyhow::Result<bool> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         is_dir_impl::is_dir(path)
     }
     fn is_file(this: FileLibrary, path: String) -> anyhow::Result<bool> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
```

Comment:
Nit since this line is a throw-away to appease the linter I don’t love that it’s multiline. Is there a non toxic way to keep this single line?
Is there a better way to ignore the this variable?

Reply:
We can add an `#[allow(whatever_its_yelling_about)]` for this one part

Reply:
Let's try that.